src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        final int lottery_count = 3;
+        for (int i = 0; i < lottery_count; i++) {
+            Lottery lot = new Lottery();
```

Comment:
create lottery outside the loop. we dont need to create new lottery each time we want to get ball